agent/config/builder.go

```diff
@@ -964,6 +965,9 @@ func (b *Builder) Validate(rt RuntimeConfig) error {
 	if rt.BootstrapExpect > 0 && rt.Bootstrap {
 		return fmt.Errorf("'bootstrap_expect > 0' and 'bootstrap = true' are mutually exclusive")
 	}
+	if rt.CheckOutputMaxSize < 1 {
+		return fmt.Errorf("'check_output_max_size must be positive")
```

Comment:
This would be a little more helpful if it had a reminder for discarding check output if the user intentionally set 0.
```suggestion
		return fmt.Errorf("check_output_max_size must be positive, to discard check output use the discard_check_output flag")
```